src/osemgrep/networking/Rule_fetching.ml

```diff
@@ -80,7 +86,7 @@ let mk_rewrite_rule_ids (origin : origin) : Rule_ID.t -> Rule_ID.t =
   let opt_prefix =
     match origin with
     | Local_file fpath -> prefix_for_fpath_opt fpath
-    | Other_origin -> None
+    | _ -> None
```

Comment:
I don't feel as strongly about this one but in general I prefer to err on the side of listing out

Reply:
Logic for this one is that we only ever rewrite for local files.